examples/demo_actor/README.md

```diff
@@ -57,7 +57,7 @@ dapr run --app-id demo-actor --app-port 3000 python3 demo_actor_flask.py
 1. Run Demo client in new terminal window
 
 ```bash
-$ cd examples/demo_actor
+$ cd ./demo_actor
```

Comment:
we should remove this as well

Reply:
nope. demo_actor has demo_actor subdirectory.